Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -3940,6 +3940,9 @@ namespace AssetProcessor
         // now we can update the database with this new information:
         UpdateSourceFileDependenciesDatabase(entry);
         m_jobEntries.push_back(entry);
+
+        // Signals SourceAssetTreeModel so it can update the CreateJob duration change
+        Q_EMIT CreateJobDurationChanged(newSourceInfo.m_sourceRelativeToWatchFolder);
```

Comment:
I might be wrong, but this likely should be "Create **Jobs** Duration Changed" 
This part of the asset pipeline creates multiple jobs for a source asset per builder... maybe I am missing something?

Reply:
This signal emits when the asset processor manager invokes CreateJob functions of (possibly multiple) builders that are interested in this source asset file. In other words, the number of CreateJobs that run is the number of builders that take such source asset file as input. 
I would make the signal and slot name become `(On)CreateJobsDurationChanged()`.